### PR TITLE
Q8 PATCH /api/articles/:article_id

### PR DESCRIPTION
**Pull request by bryanrafosala**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -251,4 +251,88 @@ describe("POST /api/articles/:article_id/comments", () => {
...
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toEqual("400: Bad Request");
+      });
+  });
+
+  test("PATCH 400: Patch object is formatted incorrectly", () => {
+    return request(app)
+      .patch("/api/articles/3")
+      .send({ notVotes: 1 })
```
Comment (pippaamy) [inline __tests__/app.test.js:324]: lovely testing!

File: db/controller/comment.controller.js [hunk 2]
```diff
@@ -30,7 +30,18 @@ exports.postCommentByArticleId = (req, res, next) => {
...
       next(err)
     });
 };
+
+exports.patchArticle = (req, res, next) => {
+  const { article_id } = req.params;
+  const { inc_votes } = req.body;
+  return updateVotes(article_id, inc_votes)
+    .then((article) => {
+      res.status(200).send({ article });
```
Comment (pippaamy) [inline db/controller/comment.controller.js:42]: we have two versions of the same thing which are we using?

File: db/controller/app.controllers.js [hunk 3]
```diff
@@ -30,12 +33,24 @@ exports.fetchArticlesByID = (req, res, next) => {
...
+      res.status(200).send({ article });
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
+
+exports.patchArticle = (req, res, next) => {
+  const { article_id } = req.params;
+  const { inc_votes } = req.body;
```
Comment (pippaamy) [inline db/controller/app.controllers.js:48]: ??? which are we using

**Review by pippaamy — COMMENTED** [review]
Great job  - you have a duplicate of your controller I would get rid of your spare one :)

File: db/controller/app.controllers.js (hunk 3 above)
Comment (bryanrafosala) [reply db/controller/app.controllers.js:48]: we are using the app controller one!

**Pull request merged**